### DISCUSSION TITLE
How to send FormData with files via h3 server enpoint to targeted server?

### DISCUSSION THREAD

**Discussion by vladddbbb** [nuxt/nuxt#17361] [Questions]
How to send FormData via server API or parse it correctly at least? It seems like h3js endpoint don't recognize incoming formData-body whereas direct request from a component to targeted server passes without errors. Appreciate for help!

**Comment by wobsoriano** [comment]
https://github.com/unjs/h3/issues/43
https://github.com/unjs/h3/issues/40
